app/routes/__index/quest2.tsx

```diff
@@ -0,0 +1,58 @@
+import PlusIconWhite from "~/components/Icons/PlusIcon";
+import { LinkFooter } from "~/components/Buttons/LinkButton";
+import Carousel from "~/components/Quest/Carousel";
+import { db } from "~/utils/db.server";
+import { useLoaderData } from "@remix-run/react";
+import { json } from "@remix-run/node";
+
+
+
+
+export async function loader() {
+    const allProjects = await db.badges.findMany({
+        orderBy: [
+          {
+            token_id: 'desc',
+          }
+        ]});
+
+    return json(allProjects);
+};
+
+
+export default function Quest() {
```

Comment:
Why does the file is called quest2 and not quest?

Reply:
this is a dummy for test when I had issue I delete